src/AppBundle/Controller/UserAdminController.php

```diff
@@ -96,27 +95,13 @@ public function showUsersByDepartmentAction(Department $department)
 
     public function deleteUserByIdAction(User $user)
     {
-        // If Non-ROLE_HIGHEST_ADMIN try to delete user in other department
-        if (!$this->isGranted(Roles::ADMIN) && $user->getDepartment() !== $this->getUser()->getDepartment) {
-            throw new BadRequestHttpException();
-        }
-        try {
-            // This deletes the given user
+        if ($this->isGranted(ROLES::ADMIN) || $user->getDepartment() == $this->getUser()->getDepartment()) {
```

Comment:
Okey, da prøver jeg det. Slik det er nå, er 'delete' knappen kun synlig dersom man har tilgang til å slette brukeren. Dvs, synlig overalt for HIGHEST-ADMIN, kun synlig for ditt department dersom du er SUPER-ADMIN, ikke synlig i det hele tatt for ADMIN.